web/src/app/users/UserDetails.js

```diff
@@ -198,6 +198,8 @@ export default function UserDetails(props) {
       <DetailsPage
         title={user.name + (svcCount ? ' (On-Call)' : '')}
         details={user.email}
+        noMarkdown
+        detailsStyles={{ marginBottom: '1rem' }}
```

Comment:
Instead of adding a new prop for this, can we bake it into the `DetailsPage` and just fix the margin when `noMarkdown` is set, automatically? The caller of `DetailsPage` shouldn't need to handle/override internal styles when using supported props.